docs/nodes-and-miners/miner-testnet.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
see the comment above - it's the same docs, just for testnet rather than mainnet

Reply:
Leaving this for now per our conversation, with the plan to copy/paste after initial fixes and then find a better solution.